packages/shared/src/components/modals/SubmitArticleModal.tsx

```diff
@@ -182,7 +182,7 @@ export default function SubmitArticleModal({
   return (
     <Modal
       {...modalProps}
-      kind={Modal.Kind.FlexibleCenter}
```

Comment:
Why did this one change?

Reply:
Pretty sure they should both be FlexibleCenter right?

Reply:
Their height changes based on the content so it causes them to shift, that's why I changed them both. in the design one of they were stated as flexible top:

https://www.sketch.com/s/2867ec63-0966-4647-b1d7-b6fe696ba251/a/dlpl9AZ